src/main/scala/scala/meta/serialiser/mappable.scala

```diff
@@ -45,6 +45,12 @@ class mappable extends StaticAnnotation {
     object FromMap {
       val ctorValuesName: Term.Name = q"values"
 
+      // get default value and store those value as a map in object
+      val defaultValue = paramss.flatten collect {
```

Comment:
for readability's sake I'd suggest `val defaultValues: Seq[Term.ApplyInfix] = ...` here.. what do you think?